### ISSUE TITLE
Shortcode button is 

### ISSUE THREAD

**Issue by ericrrichards** [OrchardCMS/OrchardCore#13105]
### Describe the bug
With the Wysiwyg editor for a MarkdownBody content part, the Shortcodes button is higher than other buttons.
![image](https://user-images.githubusercontent.com/2787661/213060019-3897f3ce-f621-4f02-adc1-b2651f456b29.png)
The CSS for this button gives it a top of -5px, which appears to be the opposite of what it should be to align correctly (+5px, maybe +7px)
![image](https://user-images.githubusercontent.com/2787661/213060316-2968b8ce-cb8d-4550-86bd-9230e720448b.png)

### To Reproduce
1. Use Orchard 1.5.0
2. Create a Content Type with a Markdown Body part
3. Create an instance of the new content
4. Observe the editor

### Expected behavior
![image](https://user-images.githubusercontent.com/2787661/213060519-89171846-d0e8-4c7e-b999-5a3dc18a60f0.png)

**Comment by hishamco** [comment]
Ya, this is a valid bug I saw it when I looked into #13050, but I forgot to file an issue

**Comment by agriffard** [comment]
Thank you, see also https://github.com/OrchardCMS/OrchardCore/pull/12481#issuecomment-1256701284

**Comment by hishamco** [comment]
@agriffard is this will be fixed in your PR? or just a normal upgrade

**Comment by agriffard** [comment]
> @agriffard is this will be fixed in your PR? or just a normal upgrade

No, I had the problem in my PR as you can see on the screenshot so it has not been merged yet.

**Comment by hishamco** [comment]
Hope to find time to fix it
